client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+            "selectGotoTarget": "Setzte einen Zielpunkt:",
+            "selectZones": "Auswahl der Zonen zur Reinigung:",
+            "selectMapToLoad": "Auswahl des Speicherplatzes zum Laden der Karte:",
+            "emptyZoneChosen": "Es wurde eine leere Zone gewählt. Bitte zuerst die Zone einrichten.",
+            "tooManyZones": "Es wurde versucht {{count}} Unterzonen zu wählen. Es werden maximal 5 unterstützt.",
+            "chooseMapName": "Titel zum Speichern der Karte wählen:",
+            "invalidMapName": "Der Titel darf nur Buchstaben, Zahlen, Leerzeichen und Bindestriche enthalten.",
+            "slotAlreadyTaken": "Speicherplatz #{{slot}} ist bereits in Benutzung",
+            "nameAlreadyTaken": "Name \"{{name}}\" wurde schon verwendet",
+            "noMapsStored": "Keine gespeicherte Karten!",
+            "slotNo": "Speicherplatz #",
+            "confirmMapDelete": "Soll \"{{name}}\" wirklich gelöscht werden?",
+            "wait": "Warten",
+            "waitMapTicks": "Kartenersetzungsroutinen laufen. Bitte weitere {{seconds}} Sekunden warten...",
+            "waitMapFinished": "Beendet",
+            "waitMapEncourage": "Angeforderte Routinen sollten nun beendet sein.",
+            "mapStoredOK": "Karte erfolgreich gespeichert!",
+            "mapDeletedOK": "Gespeicherte Karte erfolgreich gelöscht!",
+            "mapRestoredOK": "Karte erfolgreich wiederhergestellt!"
+        },
+        "fanspeedPresets": {
+            "whisper": "Flüstern",
+            "quiet": "Leise",
+            "balanced": "Ausgeglichen",
+            "turbo": "Turbo",
+            "max": "Maximal",
+            "mop": "Wischen"
+        }
+    },
+    "map": {
+        "status": "Status",
+        "battery": "Batterie",
+        "connecting": "Verbinden...",
+        "statusText": "{{status, lowercase}}",
+        "errorText": "{{error, lowercase}}",
+        "confirmFullCleaning": "Zonenreinigung kann nicht gestartet werden: Keine Zonen spezifiziert. Soll stattdessen eine komplette Reinigung gestartet werden?",
+        "cannotResume": "Nichts zur Wiederaufnahme!",
+        "tooManyZones": "Es können nicht mehr als 5 Zonen auf der Karte gesetzt werden."
+    },
+    "zones": {
+        "headerZones": "Eingestellte Zonen",
+        "headerGotoLocations": "Eingestellte Zielpositionen",
+        "headerForbiddenMarkers": "Verbotsmarkierungen",
+        "noZonesConfigured": "Bisher wurden keine Zonen eingestellt.",
+        "noSpotsConfigured": "Bisher wurden keine Zielpunkte eingestellt.",
+        "editForbiddenZones": "Einstellen von Verbotszonen",
+        "noPersistentData": "Dauerhafte Daten sind abgeschaltet oder nicht verfügbar. Es können keine Verbotsmarkierungen bearbeitet werden.",
+        "addButton": "Hinzufügen",
+        "renameButton": "Umbenennen",
+        "deleteButton": "Löschen",
+        "newZoneName": "Zonenname spezifizieren ...",
+        "newSpotName": "Markierungsname spezifizieren...",
```

Comment:
to avoid mixing this up with the forbidden zone markers that are partially translated with "Markierung" and keep the reference to the "spot" button on the home page I suggest to use "Spotreinigungspunkt" or keep "Spot" in the name:
"enterSpotName": "Spotnamen festlegen..."